src/global/Loading.vue

```diff
@@ -1,31 +1,39 @@
 <template>
-  <v-progress-linear :indeterminate="!!loading" color="secondary" height="10" background-opacity="0" />
+  <v-progress-linear :indeterminate="loading > 0" color="secondary" height="10" bg-opacity="0" />
 </template>
 
 <script>
-export default {
-  data: () => ({
-    loading: 0,
-  }),
-  beforeCreate() {
-    this.$API.interceptors.request.use((config) => {
-      this.loading++
+import { ref, onBeforeUnmount } from 'vue'
+import api from '@/plugins/api'
 
+export default {
+  name: 'LoadingIndicator',
```

Comment:
Does it matter if the filename doesn't match the component name? (`Loading` vs `LoadingIndicator`)

Reply:
Ahh, I see where it's mapped in `src/global/components.js` 👍

Reply:
The linter was requiring two word names, and I was trying to keep changes to a minimum so didn't want to rename the files.

Reply:
It looks like I've missed mapping components not in global though. It doesn't seem to matter for some reason.

Reply:
After a bit of research names aren't required, but sometimes useful for debugging and errors. I was only trying to get the linter to be quiet.